app/components/forms/wizard/sessions-speakers-step.js

```diff
@@ -181,6 +185,13 @@ export default Component.extend(EventWizardMixin, FormMixin, {
         this.sendAction('save');
       });
     },
+    unpublish() {
+      this.onValid(() => {
+        this.set('data.event.state', 'draft');
+        this.sendAction('save', this.data);
+    });
```

Comment:
Codacy found an issue: [Expected indentation of 6 spaces but found 4.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6140894)